src/Sql/Sql.Test/ScenarioTests/ManagedDatabaseBackupTests.ps1

```diff
@@ -0,0 +1,176 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+	.SYNOPSIS
+	Test short term retention for managed databases.
+#>
+function Test-ManagedLiveDatabaseShortTermRetentionPolicy
+{
+	# Setup
+	$rg = Create-ResourceGroupForTest
+	$vnetName = "cl_initial"
```

Comment:
I tried to do this, however I found out that there are a lot of parameters hard coded for MI functions in "Common.ps1". Correct thing to do would be to change the common functions to use the getAssetName, instead of hard coded values. However this would mean that I have to rerecord all of the MI tests and since we are creating new v-net for each tests, it would take roughly 4 hours per test. Complete change would take days even if there are no issues with tests. I would like to avoid this procedure if possible :).

Reply:
Okay we can worry about this at a later point.